react-app/public/locales/en/translation.json

```
@@ -0,0 +1,53 @@
+{
+  "welcome": "Welcome to YT Plugin",
+  "loginByGoogle": "Login by Google",
+  "savedKeywords": "Saved Keywords",
+  "foundVideos": "Found Videos",
+  "search": "Search",
+  "addKeyword": "Add keyword",
+  "keywordErrors": {
+    "duplicated_keyword": "This keyword already exists",
+    "keywords_not_found": "Keyword not found"
```

```suggestion
    "keywords_not_found": "Keywords not found"
```

Cause you try to fetch many